pkg/lang/python/plugin_expose.go

```diff
@@ -210,23 +210,23 @@ func (h *restAPIHandler) handleFile(f *core.SourceFile) (*core.SourceFile, error
 			continue
 		}
 
-		appName = app.Identifier.Content(f.Program())
+		appVarName = app.Identifier.Content(f.Program())
 		h.RootPath = app.RootPath
 
 		gwSpec := gatewaySpec{
 			FilePath:   f.Path(),
-			AppVarName: cap.ID,
+			AppVarName: appVarName,
```

Comment:
I think this is going to break the resource validation though. iirc, the resource validation is looking for a gateway with a given gateway ID as the name

Reply:
So that might have been why this became an issue, the name was changed to fix the resource validation issue but then caused a regression w/ proxy calls

Reply:
yeah same comment as below